sd_jwt/bin/sd_jwt

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import logging
+import random
+import sys
+
+from jwcrypto.jwk import JWK
+from sd_jwt import __version__
+from sd_jwt.operations import (
+    create_sd_jwt_and_svc,
+    create_release_jwt,
+    generate_salt,
+    verify
+)
+from typing import Union
+
+logger = logging.getLogger(__name__)
+
+
+def print_repr(values: Union[str,list], nlines=2):
```

Comment:
I'd move these two function in a `utils.py` file

Reply:
Done here
https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/0b17a2be43a62af6a02ebd9df220a77543d32748